docs/usage/syncthing/index.rst

```diff
@@ -0,0 +1,426 @@
+===========================
+Syncthing-based replication
+===========================
+
+.. toctree::
+   :hidden:
+
+   syncthing_example
+
+.. sidebar:: Contents
+
+   .. contents:: Syncthing-based replication
+      :local:
+
+VolSync supports synchronization of PersistentVolume data across a vast number of volumes using a Syncthing-based data mover.
+
+ReplicationSource objects are configured to connect to other Syncthing devices in order to sync data of a provided PVC.
+Any changes made to the PVC will be propagated to the rest of the peers sharing the volume.
+
+------
+
+
+
+How Syncthing Works In VolSync
+==============================
+
+Syncthing connects to a cluster of other Syncthing devices that are all sharing the same synchronized volume.
+When one of the devices changes data, it will propagate the change to all the other devices.
+Syncthing also includes an introducer feature which allows one device to be connected to a cluster of other devices upon configuring a single introducer node.
+This can be used to create a hub-and-spoke model for replication, or any other kind of network.
+
+When a ReplicationSource is created and configured to sync a PVC with other peers, all of the connected peers will maintain their
+own Volume containing the synced data. To detect file changes, Syncthing employs two methods: a filesystem watcher, which notifies 
+Syncthing of any changes to the local filesystem, and a full filesystem scan which occurs routinely at a specified interval (default is an hour).
+Syncthing itself is an "always-on" synchronization system, and so created ReplicationSources will never finish their syncs unless deleted.
+
+VolSync uses a custom-built Syncthing mover which disables the use of relay servers and global announce, and relying instead on 
+being provided with the addresses of other Syncthing peers directly.
+
+
+.. note::
+    Syncthing is peer-to-peer technology which connects to other peers directly rather than going through intermediary servers.
+    Because Syncthing lacks centralization, file conflicts are resolved by favoring the `most recent version <https://docs.syncthing.net/users/syncing.html#conflicting-changes>`_.
+
+
+Communicating With Syncthing
```

Comment:
Perhaps move this to an advanced configuration section? I don't think users would normally do this, correct?